optin/reporting/test/models/completed.spec.js

```diff
@@ -0,0 +1,204 @@
+const runDBNotFoundTests = require('./runDBNotFoundTests');
+const runTotalRecordTests = require('./runTotalRecordTests');
+const Config = require('../../config');
+const completedDB = require('../../models/completed');
+const { refreshDBs } = require('../../models/database');
+const { expect } = require('chai');
+const { stub, restore } = require('sinon');
+const path = require('path');
+
+const runBasicOptinTypeTests = (types, parameters) => {
+	/* eslint-disable no-await-in-loop */
+	for (let i = 0; i <= types.length - 1; ++i) {
```

Comment:
can you use forEach here?

Reply:
I did try that, but `forEach` is not working for `await/async`.

https://stackoverflow.com/questions/37576685/using-async-await-with-a-foreach-loop

I can change it to `for … of` if this looks better.

Reply:
it's ok, you can leave it